src/com/spikes2212/motor/Gearbox.java

```diff
@@ -7,59 +7,60 @@
 
 import java.util.List;
 
-public class Gearbox implements SpeedController {
+public class Gearbox extends WPI_TalonSRX{
 
-    private WPI_TalonSRX master;
     private List<SpeedController> victorSPs;
+    private int port;
 
-    public Gearbox(WPI_TalonSRX master) {
-        this.master = master;
+    public Gearbox(int port) {
+        super(port);
+        this.port = port;
     }
 
     @Override
     public void set(double speed) {
-        master.set(speed);
+        this.set(speed);
     }
 
     @Override
     public double get() {
-        return master.get();
+        return this.get();
     }
 
     @Override
     public boolean getInverted() {
-        return master.getInverted();
+        return this.getInverted();
     }
 
     @Override
     public void setInverted(boolean isInverted) {
-        master.setInverted(isInverted);
+        this.setInverted(isInverted);
     }
 
     @Override
     public void disable() {
-        master.disable();
+        this.disable();
     }
 
     @Override
     public void stopMotor() {
-        master.stopMotor();
+        this.stopMotor();
     }
 
     @Override
     public void pidWrite(double output) {
-        master.pidWrite(output);
+        this.pidWrite(output);
     }
 
     public void enslaveTalonSRX(WPI_TalonSRX... slaves) {
```

Comment:
please replace these two `enslave` methods with one that receives [`BaseMotorController`](http://www.ctr-electronics.com/downloads/api/java/html/classcom_1_1ctre_1_1phoenix_1_1motorcontrol_1_1can_1_1_base_motor_controller.html#ad4f61f2782b6048c7f905bbf38b066ac) instances, so it is more generic.